### PR TITLE
[CI] [GHA] Download `ninja` from GitHub instead of installing through `choco` in Windows workflows

### PR DESCRIPTION
**Pull request by akashchi**
### Tickets:
 - *CVS-131217*

### CODE REVIEW

File: .github/workflows/windows_conditional_compilation.yml [hunk 1]
```diff
@@ -109,7 +109,10 @@ jobs:
           version: "v0.5.4"
```
Comment (ilya-lavrenov) [inline .github/workflows/windows_conditional_compilation.yml:109]: interesting why sccache version is different in Windows CC..

**Review by ilya-lavrenov — APPROVED** [review]

File: .github/workflows/windows_conditional_compilation.yml (hunk 1 above)
Comment (akashchi) [reply .github/workflows/windows_conditional_compilation.yml]: Fixed.

**Review by gdamiaN538 — COMMENTED** [review]
>

**Review by gdamiaN538 — COMMENTED** [review]
*

**Pull request merged**